src/features/pages/home/live-pricing/table-component/live-market-table.tsx

```diff
@@ -33,41 +41,61 @@ const LiveMarketTable = ({ selected_market, link_to }: TLiveMarketTableProps) =>
 
     const [sorting, setSorting] = React.useState<SortingState>([])
 
-    const region = useMemo(() => {
-        return is_eu
-            ? 'https://deriv-static-pricingfeed.firebaseio.com/eu.json'
-            : 'https://deriv-static-pricingfeed.firebaseio.com/row.json'
-    }, [is_eu])
+    const region = is_eu
+        ? 'https://deriv-static-pricingfeed.firebaseio.com/eu.json'
+        : 'https://deriv-static-pricingfeed.firebaseio.com/row.json'
 
-    const getData = useCallback(async () => {
-        const rawResponse = await fetch(region)
-        const response = await rawResponse.json()
-        setRawMarketsData(response)
-    }, [region])
+    useEffect(() => {
+        if (typeof window !== 'undefined') {
+            window.addEventListener('online', () => setIsOffline(false))
+            window.addEventListener('offline', () => setIsOffline(true))
+        }
+    }, [])
```

Comment:
What do you say if instead of initial checking we'll check that user'll  online/offline each 5 seconds?
```suggestion
    }, 5000)
```

Reply:
@NikitK-deriv This issue is only for initial loading. After the initial load, if the network is slow or user goes offline it will stop fetching and displays previous data.